runtime/interpreter/decode.go

```diff
@@ -196,360 +110,221 @@ func (d *DecoderV5) decodeValue(path []string) (Value, error) {
 
 		switch num {
 
-		case cborTagVoidValue:
+		case atree.CBORTagStorageID:
+			return atree.DecodeStorageIDStorable(d.decoder)
+
+		case CBORTagVoidValue:
 			err := d.decoder.Skip()
 			if err != nil {
 				return nil, err
 			}
-			value = VoidValue{}
-
-		case cborTagDictionaryValue:
-			value, err = d.decodeDictionary(path)
+			storable = VoidValue{}
 
-		case cborTagSomeValue:
-			value, err = d.decodeSome(path)
-
-		case cborTagAddressValue:
-			value, err = d.decodeAddress()
+		case CBORTagStringValue:
+			v, err := d.decoder.DecodeString()
```

Comment:
As mentioned above Cadence string values are now encoded with a CBOR tag, not just as plain CBOR strings.